core/app/models/spree/in_memory_order_updater.rb

```diff
@@ -130,9 +130,9 @@ def recalculate_adjustments(persist:)
     # +promo_total+        The total value of all promotion adjustments
     # +total+              The so-called "order total."  This is equivalent to +item_total+ plus +adjustment_total+.
     def update_totals(persist:)
-      update_payment_total
-      update_item_total
-      update_shipment_total
+      recalculate_payment_total
+      recalculate_item_total
+      recalculate_shipment_total
```

Comment:
these should also be deprecated because even though they are private we all have evidence that this are overwritten in subclasses. can be extracted into a PR